web-api/src/trialSessions/addCaseToTrialSessionLambda.js

```diff
@@ -16,5 +16,6 @@ exports.addCaseToTrialSessionLambda = event =>
         applicationContext,
         docketNumber,
         trialSessionId,
+        ...JSON.parse(event.body),
```

Comment:
And please don't change anything in this PR, but just 🤔  curious if you'd consider using `joi` to perform initial simple validation on the requests here -- or just leave it on the entity validation later in the app.

Reply:
That's an interesting idea. I know that we've written some code in the past that essential is an allow list for fields on the interactor side - so regardless what you throw at it, it will only use those explicit fields. I'm going to log this as a DevEx item to discuss with the team! Thanks!